charts/docker-hub-rate-limit-exporter-chart/templates/secret.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   labels:
     {{- include "docker-hub-rate-limit-exporter.labels" . | nindent 4 }}
 data:
-  dockerhub-password: {{ default "" .Values.config.dockerhubPassword | b64enc | quote  }}
\ No newline at end of file
+  dockerhub-password: {{ .Values.config.dockerhub.password | b64enc | quote  }}
```

Comment:
this is another breaking change which improves readability but is not entirely necessary. I think to make it easier for users to upgrade to 0.3.0 I would prefer not to make that change. 

But I can also see the benefits. I leave it to you if you want to revert to `config.dockerhubPassword` or keep your change.

Reply:
Same goes for `.Values.config.dockerhubUsername`.